x-pack/test/functional/apps/ml/data_frame_analytics/classification_creation.ts

```diff
@@ -96,9 +96,9 @@ export default function ({ getService }: FtrProviderContext) {
           await ml.dataFrameAnalyticsCreation.continueToAdditionalOptionsStep();
         });
 
-        it('inputs the model memory limit', async () => {
+        it('accepts the suggested model memory limit', async () => {
```

Comment:
Nit: should we use "should" like `it('should accept the suggested model memory limit', ...`?